src/deviceinfo.h

```diff
@@ -220,11 +222,42 @@ class SYSTEMSETTINGS_EXPORT DeviceInfo: public QObject
      */
     QString adaptationVersion() const;
 
+    /*!
+     * List of IMEI number strings
+     *
+     * Note: Access is limited to users in
+     * \l {https://github.com/sailfishos/sailfish-setup} {sailfish-phone}
+     * group. Additionally sandboxed applications need
+     * \l {https://github.com/sailfishos/sailjail-permissions} {Phone}
+     * permission. If these requirements are not met, an empty list
+     * is returned.
+     *
+     * Obtained by enumerating modems exposed on D-Bus by OFono service.
+     *
+     * Normally enumeration is done asynchronously in background and
+     * empty list is returned until enumeration is finished.
+     *
+     * If DeviceInfo object is created using synchronousInit=true,
+     * constructor blocks while getting the initial values - which may
+     * be useful when dealing with procedural logic.
+     *
+     * If/when OFono is not running, an empty list is returned regardless
+     * of whether asynchronous or synchronous init is used.
+     *
+     * Should be functionally equivalent with obtaining data via:
+     *   QDeviceInfo::imei()
+     *   QDeviceInfo::imeiCount()
```

Comment:
Could myself skip this latter notice as imeis should be equivalent no matter where they are gotten and the usage isn't too widespread. But doesn't matter much.